.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
This `.gitignore` file is a special file that tells git to completely ignore certain kinds of files. The editor I was using, InteliJ IDEA has a special folder that has a bunch of configurations. So the folder structure looks like
```
Pui_Assignment-3
    - myscript.js
    - .gitignore
    // etc etc and all of your code
    - .idea        // the IntelliJ folder with information about the project
        // a bunch of random IntelliJ files
    - .git           // the folder that has a bunch of information that git uses
```

And it doesn't make sense for you do have the stuff in the `.idea` folder, so now git won't track changes to those files, and it won't get added to Github when I push

(the `.` before a folder means it's a hidden folder. So if you look for it in Finder you won't see it)